config/src/config/state_sync_config.rs

```diff
@@ -1,8 +1,16 @@
 // Copyright (c) Aptos
 // SPDX-License-Identifier: Apache-2.0
 
+use crate::config::MAX_APPLICATION_FRAME_SIZE;
 use serde::{Deserialize, Serialize};
 
+/// We allow a buffer in case network messages get slightly larger after they
+/// are checked against the max frame size but before they are actually sent to
+/// the networking stack. This can occur in specific cases, e.g., if the
+/// message is wrapped in an another type before being sent down the wire. We
+/// should rarely see this in practice (if ever), but let's play it safe.
```

Comment:
Let's keep in mind that we should be brief in our comments, because it gets very long.